src/zip/mod.rs

```diff
@@ -427,6 +430,25 @@ where
     }
 }
 
+impl<D, P1, P2> Zip<(P1, P2), D>
+    where D: Dimension,
```

Comment:
The formatting of this `where` block doesn't match how the `where` blocks for the other `impl`s are formatted.